packages/playwright/README.md

```diff
@@ -61,3 +73,10 @@ This is very much underdeveloped/WIP. At the moment, there exists a (git-shared)
 file which can be used for configuring certain test attributes. This is most likely
 about to change in the future when the test suite is supposed to be used in, e.g.,
 GitHub Actions pipelines. Stay tuned for updates!
+
+## Troubleshooting tips
+
+On MacOS, you might run into the following error:
```

Comment:
I haven't seen that error myself, but I think mentioning it here doesn't hurt. Maybe it's due to te monorepo setup? Not sure myself. Maybe we can/have to explicitly declare the browser version(s) in the dependencies...